app/components/RadioPlayer.vue

```diff
@@ -533,10 +539,8 @@ export default {
             text-overflow: ellipsis;
             overflow: hidden;
             white-space: nowrap;
```

Comment:
@molnar-a yeah I also noticed it in the browser emulator but it seems to render well on real devices. The key property is `white-space` here and in the code it's clear that it should be `nowrap` over 380px (this is what the simulator is doing too) but it was OK (wrapped) on all real devices and mobile browsers I checked. So yes, the code may not be ideal here but as long as it seems to work in the real world I wouldn't bother touching it. I suggest that you check on your phone too and if it's fine (wrapped) then we move on with this code for now.

Reply:
fair enough! approved :)